xcode/Subconscious/Shared/Components/Common/Profile/UserProfileView.swift

```diff
@@ -359,6 +368,46 @@ private struct FollowModifier: ViewModifier {
     }
 }
 
+private struct RenameModifier: ViewModifier {
```

Comment:
Note that `private` here means "private to my project", not "private to this file".

- Recommend moving this to a separate file, or into `FollowUserSheet`
- Recommend giving it specific name, perhaps `RenameSheetModifier`

Reply:
Do you think this applies to the other modifiers in this file too? Should we break them all out? I was trying to avoid polluting the namespace with view modifiers so I actually would almost prefer `fileprivate` and keeping them at the use-site.

I'll try moving things around and see where we land. Agreed RE: the naming of the modifier.

Reply:
> Note that private here means "private to my project", not "private to this file".

Based on my testing this isn't true, it means "private to the containing lexical scope" which in this case _is_ the file. `private` at the root of a file is equivalent to `fileprivate`, no other file can access the type.

https://forums.swift.org/t/calling-private-methods-from-extensions-in-separate-files/54029

```swift
// A.swift

fileprivate struct One {}
private struct Two {}
internal Three {}
struct Four {} // implicitly `internal` by default
public struct Five {}
```

```swift
// B.swift

let one: One? = nil // Error: Cannot find 'One' in scope
let two: Two? = nil  // Error: Cannot find 'One' in scope
let three: Three? = nil
let four: Four? = nil
let five: Five? = nil
```

Reply:
I've moved all these sheets into `UserProfileView+SheetModifier` and clarified the naming convention.

Reply:
Huh, when trying to use something marked as `private` at the file root the error says that it's actually _`fileprivate`_. I guess internally `private` at file root === `fileprivate`.

<img width="1165" alt="image" src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/e321b08f-0a3f-466f-8631-3f62f0882059">